addons/escoria-core/game/core-scripts/save_data/esc_save_manager.gd

```diff
@@ -301,16 +321,27 @@ func load_game(id: int):
 
 		if global_value is String and global_value.empty():
 			global_value = "''"
-
-		load_statements.append(
-			ESCCommand.new("%s %s %s true" %
-				[
-					_set_global.get_command_name(),
-					k,
-					global_value
-				]
+		
+		if k.begins_with("i/"):
+			if global_value:
+				load_statements.append(
+					ESCCommand.new("%s %s" %
+						[
+							_add_inventory.get_command_name(),
+							k.trim_prefix("i/")
+						]
+					)
+				)
+		else:
+			load_statements.append(
+				ESCCommand.new("%s %s %s true" %
```

Comment:
Having a global variable of type string with more than one word makes set_global command to break, for example:
```
globals = {
"ESC_DIALOG_CHOSEN_OPTION": "¿Que te parece que haga la comida?",
...
```
Surrounding third argument with quotes solves the issue:
`ESCCommand.new("%s %s \"%s\" true" %`